packages/honeycomb-layout/package.json

```diff
@@ -10,19 +10,16 @@
     "start": "npm run start:production",
     "start:development": "cross-env NODE_ENV=development nodemon src/index.js --exec babel-node --watch src/ --ext js",
     "start:production": "cross-env NODE_ENV=production node dist/index.js",
-    "test": "npm run lint && npm run test:unit",
```

Comment:
sure we don't want any tests for honeycomb-layout?
we won't see anything breaking through pull-request or dependency updates